lab-Marshall/lib/cowsay.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const moo = require('cowsay');
+
+const cowsay = module.exports = function(a) {
```

Comment:
It's best practice to give your variables descriptive names, like "str" or "input" in this case